client/me/account/main.jsx

```diff
@@ -957,6 +959,23 @@ class Account extends Component {
 				<SectionHeader label={ translate( 'Interface settings' ) } />
 				<Card className="account__settings">
 					<form onSubmit={ this.saveInterfaceSettings }>
+						{ config.isEnabled( 'dashboard/v2' ) && (
+							<FormFieldset className="account__settings-admin-home">
+								<FormLabel id="account__new-hosting-dashboard">
+									<HStack justify="flex-start">
```

Comment:
This `HStack` is unfortunately needed to make sure the text and badge baselines align nicely.